test/spec/modules/liveIntentIdSystem_spec.js

```diff
@@ -1,46 +1,52 @@
-import {liveIntentIdSubmodule, reset as resetLiveIntentIdSubmodule, storage} from 'modules/liveIntentIdSystem.js';
+import { liveIntentIdSubmodule, reset as resetLiveIntentIdSubmodule, storage } from 'modules/liveIntentIdSystem.js';
 import * as utils from 'src/utils.js';
-import {uspDataHandler} from '../../../src/adapterManager.js';
-import {server} from 'test/mocks/xhr.js';
+import { gdprDataHandler, uspDataHandler } from '../../../src/adapterManager.js';
+import { server } from 'test/mocks/xhr.js';
 
 const PUBLISHER_ID = '89899';
-const defaultConfigParams = {publisherId: PUBLISHER_ID};
-const responseHeader = {'Content-Type': 'application/json'}
+const defaultConfigParams = { publisherId: PUBLISHER_ID };
+const responseHeader = { 'Content-Type': 'application/json' }
 
-describe('LiveIntentId', function () {
+describe('LiveIntentId', function() {
   let pixel = {};
   let logErrorStub;
-  let consentDataStub;
+  let uspConsentDataStub;
+  let gdprConsentDataStub;
   let getCookieStub;
   let getDataFromLocalStorageStub;
   let imgStub;
 
-  beforeEach(function () {
+  beforeEach(function() {
     imgStub = sinon.stub(window, 'Image').returns(pixel);
```

Comment:
I think we shuold replace this with a stub of`triggerPixel`

Reply:
changed to use the `triggerPixel` and added a missing test for the pixel https://github.com/LiveIntent/Prebid.js/pull/3/commits/9d55da7d211111d68592d9f01985a506f989ee04